x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/overview.test.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// import { act } from 'react-dom/test-utils';
+
+import { mockKibanaSemverVersion } from '../../../common/constants';
+import { OverviewTestBed, setupOverviewPage, setupEnvironment } from '../helpers';
+
+describe('Overview Page', () => {
+  let testBed: OverviewTestBed;
+  const { server } = setupEnvironment();
+
+  beforeEach(async () => {
+    testBed = await setupOverviewPage();
+    testBed.component.update();
+  });
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  describe('Documentation links', () => {
+    test('Has a whatsNew link and it references nextMajor version', () => {
+      const { exists, find } = testBed;
+      const nextMajor = mockKibanaSemverVersion.major + 1;
+
+      expect(exists('whatsNewLink')).toBe(true);
+      expect(find('whatsNewLink').text()).toContain(`${nextMajor}.0`);
+    });
+
+    test('Has a documentation link', () => {
```

Comment:
[nit] Can we be more specific here (the describe block is also called `Documentation links`)?